101-acs-swarm/azuredeploy.parameters.json

```diff
@@ -6,7 +6,7 @@
       "value": "GEN-UNIQUE"
     },
     "agentCount": {
-      "value": "1"
+      "value": 1
```

Comment:
@rgardler, actually this is fine. The metdata file is invalid JSON. @stuartleeks Can you check?